### ISSUE TITLE
State and actions not showing in Flipper app

### ISSUE THREAD

**Issue by denispixi** [zrg-team/flipper-rn-redux-inspector-plugin#4]
In Flipper the plugin is installed and enabled, and in my iOS app, rn-redux-middleware-flipper and  react-native-flipper are installed too, but in the flipper app on the RN Redux Inspector section nothing is showing 

![image](https://user-images.githubusercontent.com/20715348/81479867-b6e4a000-91eb-11ea-85c6-f87e08458c82.png)

What i'm missing or misconfigured?

**Comment by zrg-team** [comment]
@denispixi de
did you implement redux's plugin ? 
```
if (__DEV__) { // eslint-disable-line
  const createFlipperMiddleware = require('rn-redux-middleware-flipper').default
  middlewares.push(createFlipperMiddleware())
}
```

**Comment by denispixi** [comment]
Yes, I did that. I figured out that nothing is shown until some redux action is dispatched. Also this: if I leave the plugin section in flipper (go to RN logs for example), do some updates to the store and return to RN-redux-inspector the state is not up-to-date, it get updated only if the plugin is focused when some action is dispatched
